testgrid/config.yaml

```diff
@@ -6092,6 +6098,10 @@ dashboards:
     test_group_name: pull-cluster-api-provider-aws-bazel-integration
   - name: test-creds
     test_group_name: periodic-cluster-api-provider-aws-test-creds
+  - name: periodic-bazel-e2e
+    test_group_name: periodic-cluster-api-provider-aws-bazel-e2e
+  - name: pr-bazel-e2e
```

Comment:
To match the other names (bazel-pr-verify, bazel-pr-build, etc), could we call this bazel-pr-e2e?